tests/system/small/test_multiindex.py

```diff
@@ -16,7 +16,37 @@
 import pytest
 
 import bigframes.pandas as bpd
-from tests.system.utils import assert_pandas_df_equal
+from tests.system.utils import assert_pandas_df_equal, skip_legacy_pandas
+
+
+@skip_legacy_pandas
+def test_read_pandas_multi_index_axes():
+    index = pandas.MultiIndex.from_arrays(
+        [
+            pandas.Index([4, 99], dtype=pandas.Int64Dtype()),
+            pandas.Index(
+                ["Hello, World!", "some_new_string"],
+                dtype=pandas.StringDtype(storage="pyarrow"),
+            ),
+        ],
+        names=["1index 1", "1index 2"],
+    )
+    columns = pandas.MultiIndex.from_arrays(
+        [
+            pandas.Index([6, 87], dtype=pandas.Int64Dtype()),
+            pandas.Index(
+                ["Bonjour le monde!", "une chaîne de caractères"],
+                dtype=pandas.StringDtype(storage="pyarrow"),
+            ),
+        ],
+        names=["1columns 1", "1new_index 2"],
+    )
+    pandas_df = pandas.DataFrame(
+        [[1, 2], [3, 4]], index=index, columns=columns, dtype=pandas.Int64Dtype()
+    )
+    bf_df = bpd.DataFrame(pandas_df)
```

Comment:
Are we sure pandas itself doesn't do any normalization on the labels? Can we assert that before this conversion?

Reply:
pandas directly uses the passed index object without modification.